src/main/java/frc/robot/utilities/pid/PIDConfigurable.java

```diff
@@ -9,7 +9,11 @@
 
     double getSetpoint();
 
-    void setSetpoint(double setpoint);
+    void setSetpoint(double setpoint, boolean isTuning);
```

Comment:
because the point of this `isTuning` is to check whether the caller of the function meant to change the setpoint while tuning in purpose, or it's a command, so we'll know to ignore the call.